lab_4/lab_4.cpp

```diff
@@ -0,0 +1,97 @@
+#include <iostream>
+using namespace std;
+
+void power(int digit, int pow) {
+	int pow, i = 0;
+	float b = 1.0;
+	if (digit != 0 && pow != 0) {

```

Comment:
Why user can't use 0 as power?

Reply:
0 to the power 0 as a special case, uncertainty, for all other numbers this power works